docs/assets/scss/docs.scss

```diff
@@ -432,3 +432,26 @@ a[data-open-video] {
     color: #aaa;
   }
 }
+
+.button.clear {
```

Comment:
Clear buttons are coming, so we don't want to conflict... lets scope this to docs & add comment that this will come in the next release and these styles should be removed then.